src/scss/reviews.scss

```diff
@@ -0,0 +1,123 @@
+.reviews__section {
+  background: #081130;
+
+  padding: 80px 0;
+}
+
+.reviews__title {
+  margin-bottom: 50px;
+
+  color: #ffffff;
+  text-align: center;
+}
+
+.user-card {
+  display: flex;
+
+  padding: 70px 140px;
+
+  background-color: #ffffff;
+  box-shadow: 2px 8px 16px rgba(194, 198, 208, 0.06);
+  border: 2px solid #E2EAF2;
+  border-radius: 5px;
+}
+
+.user-card-review {
+  width: 60%;
+  margin: 0 54px 0 0;
+}
+
+.user-card-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: 40%;
+}
+
+.user-card-review__title {
+  display: flex;
+  flex-direction: column;
+
+  margin: 0 0 24px 0;
+
+  color: #2E363E;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 143%;
+
+  &:before {
+    content: "❝";
+
+    margin: 0 0 24px 0;
+
+    font-size: 5rem;
+    color: #5D6AFB;
+  }
+}
+
+.user-card-review__text {
+  margin: 30px 0 24px 0;
+
+  color: #2E363E;
+  font-size: 15px;
+  line-height: 170%;
+}
+
+.user-card-review__link {
+  font-size: 14px;
+  line-height: 21px;
+  color: #4A94FF;
+  border-bottom: 1.5px dashed #4A94FF;
+}
+
+.user-card-info__title {
+  color: #2E363E;
+  font-size: 21px;
+  font-weight: 500;
+  line-height: 143%;
+  margin-top: 30px;
+}
+
+.user-card-info__subtitle {
+  font-size: 16px;
+  line-height: 178%;
+  color: #A6AEB6;
+  font-weight: 400;
+}
+
+.reviews-arrows__container {
+  margin-top: 70px;
```

Comment:
Try to keep margins direction from up to down, that margin should be set in the upper element, and be a margin-bottom.